packages/frontend/src/api/useDialogs.tsx

```diff
@@ -29,6 +30,13 @@ interface UseDialogsOutput {
   isLoading: boolean;
 }
 
+export const getSenderName = (
+  item: SearchDialogFieldsFragment | DialogByIdFieldsFragment,
```

Comment:
Maybe you can use `getPreferredPropertyByLocale`?  Not sure why I understand this should be treated differently than content such as title and description?